Fix more windows

Kasuobes opened the pull request:
Fixed two windows in the brig by the delivery disposals and one by the bar mulebot delivery tile. Also fixed some turfs in the brig and research outpost where tiled flooring were under wingrilles instead of plating.

PsiOmegaDelta commented:
I note the patient rooms no longer have polarized windows, despite the polarized window spawner supporting being able to set the appropriate id.

Kasuobes commented:
^ Fixed.

PsiOmegaDelta commented:
Didn't set the id from what I can tell. Means the window tint buttons won't toggle the nearby windows at all.

Kasuobes commented:
Just tested ingame with existing polarized wingrilles and tint switches, they function like light switches and do not require id.

Also fixed CE's office polarizing wingrilles.

PsiOmegaDelta commented:
But if you don't set the id it potentially means the patient room buttons will toggle windows in other rooms due to the short range..

Kasuobes commented:
The patient rooms are separated into their own areas, so that shouldn't happen. At least, that's the way I think light switches work.

Kasuobes commented:
I was dead wrong. Isolation room windows now have IDs, also prevents the psyc window tint controls from blacking out half of IsoA.

Kasuobes commented:
Added more fixes.

> Readded windows in the brig by the delivery disposals, by the bar mulebot delivery area, gateway, engineering workshop, toxins lab.
Fixed polarizing wingrilles in the CE's office and medbay isolation rooms.
Fixed several turfs in the brig, research outpost and arrivals hallway, where tiled flooring were under wingrilles instead of plating.

PsiOmegaDelta commented:
:+1:
